tsl/test/expected/compression_ddl.out

```diff
@@ -54,6 +54,48 @@ DROP INDEX new_index;
 ALTER TABLE test1 SET (fillfactor=100);
 ALTER TABLE test1 RESET (fillfactor);
 ALTER TABLE test1 ALTER COLUMN b SET STATISTICS 10;
+-- ensure that REPLICA IDENTITY works
+ALTER TABLE test1 REPLICA IDENTITY FULL;
+SELECT relname, relreplident FROM pg_class WHERE relname = 'test1' ORDER BY relname;
+ relname | relreplident 
+---------+--------------
+ test1   | f
```

Comment:
Can u run it before and after changing the replica identity to make evident that it changed properly.

Reply:
done

Reply:
done